forms/tests/test_api.py

```diff
@@ -184,3 +187,27 @@ def test_answer_post(admin_client, admin_user, client, user_factory, basic_form)
     url = reverse("answer-list")
     response = client.get(url)
     assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_attachment_post(admin_client, basic_form):
+    example_file = SimpleUploadedFile(name="example.txt", content=b"Lorem lipsum")
+    payload = {
+        "field": Field.objects.all().first().id,
+        "name": fake.name(),
+        "attachment": example_file,
+    }
+    url = reverse("attachment-list")
+    response = admin_client.post(url, data=payload)
+    assert response.status_code == 201
+
+
+@pytest.mark.django_db
+def test_attachment_delete(admin_client, basic_form):
+    test_attachment_post(admin_client, basic_form)
+    attachment = Attachment.objects.all().first()
+    url = reverse("attachment-detail", kwargs={"pk": attachment.pk})
+    file_path = attachment.attachment.path
```

Comment:
would be good to assert that file exists before calling api. Just to make sure that everything works like it should and tests cant pass with false positive on last assert.